tests/acceptance/request_test.py

```diff
@@ -99,6 +114,7 @@ def test_400_if_required_body_is_missing(test_app):
     ).status_code == 400
 
 
+@pytest.mark.xfail
```

Comment:
Any reason we cannot trust it? Is a http request with conflicting content
type something we realistically need to protect against?

We can also not use the .json convenience method...
On Fri, Mar 27, 2015 at 2:13 PM Daniel Nephin <notifications@github.com>
wrote:

> In tests/acceptance/request_test.py
> <https://github.com/striglia/pyramid_swagger/pull/93#discussion_r27331662>
> :
>
> > @@ -99,6 +114,7 @@ def test_400_if_required_body_is_missing(test_app):
> >      ).status_code == 400
> >
> >
> > +@pytest.mark.xfail
>
> Ah ya, this is an annoying part of the pyramid request object. If we can
> trust the content-type header, you could add a check for that in the
> PyramidRequest object, and not actually access the POST if the context is
> json.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/striglia/pyramid_swagger/pull/93/files#r27331662>.
>